MovieSharing/VideoResponse.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  VideoResponse.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 01.10.21.
+//
+
+import Foundation
+
+struct VideosResponse: Decodable {
```

Comment:
- [ ] rename this object to something like `VideoContainer` etc. because event though, yes, this is the structure you use to store the data you fetched, this struct itself does not have any properties that relate it to a `Response`. It is more so the (decoded) data of a response